licenses/bin/HdrHistogram-2.1.8.txt

```diff
@@ -0,0 +1,27 @@
+BSD License
+
+Copyright (c) 2000-2015 www.hamcrest.org
```

Comment:
Are you sure HdrHistogram is copyrighted by hamcrest.org?

Reply:
agreed